TheMET/TheMET/FeaturedArtWorkLoadingView/FeaturedArtWorkLoadingView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  FeaturedArtWorkLoadingView.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 18/04/2023.
+//
+
+import Foundation
+import UIKit
+
+class FeaturedArtWorkLoadingView: UIView {
+    
+    static let identifier = "FeaturedArtWorkLoadingView"
```

Comment:
Bad naming. This is not an identifier, it is a XIB file name.

Reply:
fixed